spec/features/articles_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.feature "Articles", type: :feature do
+  before do
+    @category = FactoryBot.create(:category)
+  end
+
+  scenario "unsucessfully create a new article" do
+    article = FactoryBot.build(:article)
```

Comment:
This is created in each test - you could remove the repetition by pulling this out into a `let` block.

Similarly for your comment spec

Reply:
Fixed in commit d8658a9d8f9d8dd7e6a416bd13cec8bdbf25a750 - swapped the builds to a let block, but not create as they are populating the test database etc.